src/table/__integ__/resizable-columns-misc.test.ts

```diff
@@ -83,7 +79,7 @@ test(
   })
 );
 
-test(
+test.each([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20])(
```

Comment:
Running the test 20 times to ensure it is stable. I will remove it before merging.